float64 ball_distance
float64 ball_rel_goal
float64 ball_rel_image_x
float64 ball_rel_image_y

# says, whether ball was detected in top or bottom camera
# 0: top camera, 1: bottom camera
uint8 ball_in_top_or_bottom_camera

bool ball_detected_in_lastsec
bool left_marker_detected_in_lastsec
bool right_marker_detected_in_lastsec
